src/icons/Info/Info.web.js

```diff
@@ -2,25 +2,8 @@ import React from 'react';
 
 function Info(props) {
```

Comment:
`Info` is just an internal component. Won't be part of commit message header.
~~`fix(Info/react)`~~ `fix(Icon/react)`

Reply:
Sure